src/create-theme-provider.js

```diff
@@ -0,0 +1,92 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const isFunction = require('is-function');
```

Comment:
@iamstarkov I agree with @kof here. Also, this in particular is something that wouldn't add any tests. See @kof's comment above:

> Because it fixes an ancient issue with RegExp which is older than IE9, just tested typeof new RegExp('') returns an object.

So you can safely remove it w/o adding more tests